extension/src/plots/multiSource/collect.test.ts

```diff
@@ -0,0 +1,168 @@
+import { join } from 'path'
+import { collectMultiSourceEncoding } from './collect'
+
+describe('collectMultiSourceEncoding', () => {
```

Comment:
[F] This should give simple examples of how the encoding update is collected for variations in the datapoints.